gems/sorbet/lib/serialize.rb

```diff
@@ -174,7 +174,7 @@ def class_or_module(class_name)
       end
       next if denylisted_method(method)
       next if ancestor_has_method(method, Sorbet::Private::RealStdlib.real_singleton_class(klass))
-      serialize_method(method, true)
+      serialize_method(method, true, with_sig: false)
```

Comment:
If grepping is correct, it looks like these changes to pass `with_sig: false` make all the callsites pass `false`...is there any point in keeping the parameter and attendant code around?

Reply:
Nope, there's no point. We could do that as a follow up. In fact I actually made that change 9 months ago in Stripe's internal copy of this file:

http://go/ps/commit/85eb538af75